source/features/github-bugs.css

```diff
@@ -45,6 +45,7 @@
 #merge_title_field,
 #commit-summary-input, /* Restore monospace font in commit/merge title fields #5188 */
 :is(.new_public_key, .new_gpg_key) textarea { /* Monospace textareas for new SSH/GPG keys #4917 */
+	/* Same as GitHub style for `code` */
 	font-family: ui-monospace, SFMono-Regular, 'SF Mono', Menlo, Consolas, 'Liberation Mono', monospace !important;
```

Comment:
Can we stick this into a variable since we're using it in a couple of places?

Reply:
This only appears in `github-bugs.css` (which applies to the whole site) and `options.css` (which applies to the options page). Their scopes do not overlap so unfortunately no.

These aren't going to be changed frequently so it's fine to have some redundancy.